CHANGELOG.md

```diff
@@ -87,6 +87,7 @@ Here is an overview of all new **experimental** features:
 - **Kafka Scaler**: Support 0 in activationLagThreshold configuration ([#4137](https://github.com/kedacore/keda/issues/4137))
 - **Prometheus Metrics**: Expose Prometheus Metrics also when getting ScaledObject state ([#4075](https://github.com/kedacore/keda/issues/4075))
 - **Redis Scalers**: Fix panic produced by incorrect logger initialization ([#4197](https://github.com/kedacore/keda/issues/4197))
+- **Azure Queue Scaler** Fix azure queue length ([#4002](https://github.com/kedacore/keda/issues/4002))
```

Comment:
Could you move this before **Azure Service Bus Scaler:**? We shot the changelog, putting `General` on top and the alphabetically shorted, **Azure Queue Scaler** goes before **Azure Service Bus Scaler:**

Reply:
Updated. Thanks!